Dockerfile

```diff
@@ -1,8 +1,8 @@
 ###############################################################
-###         STAGE 1: Build credential-service app           ###
+###                  Build credential-service app           ###
```

Comment:
Can you please use a two-stage build like in upstream? Helps optimise Docker image size.